pages/sign-in.tsx

```diff
@@ -181,7 +187,15 @@ function SignInPage(props: any) {
           }}
         />
 
+        <div className={styles.actions} style={{ marginTop: 10 }}>
+          <input
+            type="checkbox"
+            onClick={() => setState({ ...state, adminLogin: !this.adminLogin})}
```

Comment:
I kept getting `KeyError: _this not found` until I did this change
```suggestion
            onClick={() => setState({ ...state, adminLogin: !state.adminLogin})}
```